cascade-compose/src/main/java/me/saket/cascade/Cascade.kt

```diff
@@ -118,6 +119,7 @@ fun CascadeDropdownMenu(
   offset: DpOffset = DpOffset.Zero,
   fixedWidth: Dp = 196.dp,
   shadowElevation: Dp = 3.dp,
+  tonalElevation: Dp = 3.dp,
```

Comment:
Do you want your popups to have a different shadow and tonal elevations?

Reply:
I don't, but others might and this matches the Compose API as it gives both options

Reply:
Can you point me to that? I don't see any `tonalElevation` parameter on `DropdownMenu()`:

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material/material/src/androidMain/kotlin/androidx/compose/material/AndroidMenu.android.kt;l=112;drc=cdd989b146a4758f9974b63c2e53e4eb767e961e

Reply:
Sorry wrong link, here's m3:

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material3/material3/src/androidMain/kotlin/androidx/compose/material3/AndroidMenu.android.kt;l=84;drc=15aff086d739e6673f71e5cc7bf0d01689942cc6